sdk/storage/storage-file-datalake/src/DataLakeFileSystemClient.ts

```diff
@@ -194,6 +194,12 @@ export class DataLakeFileSystemClient extends StorageClient {
         ...options,
         access: toContainerPublicAccessType(options.access),
         tracingOptions: updatedOptions.tracingOptions,
+        containerEncryptionScope: options.fileSystemEncryptionScope
+          ? {
+              defaultEncryptionScope: options.fileSystemEncryptionScope.defaultEncryptionScope,
```

Comment:
>options.fileSystemEncryptionScope.defaultEncryptionScope [](http://example.com/codeflow?start=38&length=56)

What if `defaultEncryptionScope` is undefined or empty string.

Reply:
SAA